packages/compass-connections/src/stores/connections-store-redux.ts

```diff
@@ -1657,6 +1659,14 @@ const connectWithOptions = (
           return;
         }
 
+        // We're trying to optimise the initial Compass loading times here: to
+        // make sure that the driver connection pool doesn't immediately get
+        // overwhelmed with requests, we fetch instance info only once and then
```

Comment:
Yea, in fact it doesn't really matter what we set our pool size to, browsers do not permit more than 6 parallel connections to be creating at the same time. 

It obviously matters for our total count in the long run, but at least at boot that's our bottleneck.